front/components/assistant/conversation/ConversationError.tsx

```diff
@@ -0,0 +1,33 @@
+import { Icon, XCircleIcon } from "@dust-tt/sparkle";
+
+import { isConversationAccessDeniedError } from "@app/lib/conversation";
+
+interface ConversationError {
+  error: Error;
+}
+
+export function ConversationError({ error }: ConversationError) {
+  const isAccessDeniedError = isConversationAccessDeniedError(error);
+
+  if (isAccessDeniedError) {
+    return <ConversationAccessDenied />;
+  }
+
+  return <>Error loading conversation</>;
```

Comment:
What about adding the error message?
Or at least handling the "not found" case?

Reply:
So I don't think we should expose the error in the UI. However, I did add not found error and some more style on the generic error.